en/peripherals/parachute.md

```diff
@@ -27,50 +28,85 @@ Below are a few considerations when using parachutes:
+  
+:::note
+You can also configure an [external Automatic Trigger System (ATS)](../config/safety.md#external-automatic-trigger-system-ats) for failure detection.
 :::
+  
+### Motor Failsafe Settings
+
+The motors must be set to failsafe/disarmed values.
+
+Assuming a quad setup with motors on PWM outputs MAIN1 to MAIN4 you would set [PWM_MAIN_FAILn](../advanced_config/parameter_reference.md#PWM_MAIN_FAIL1) to 900ms, where n is 1 - 4.
+This ensures that during failsafe the motors are set to disarmed values.
 
-Hardware setup:
+If using [Actuators](../config/actuators.md) configuration the failsafe values for motor outputs will be set appropriately as part of normal setup.
+
+### PWM Parachute Setup
+
+The parachute must be connected to an unused output, and the associated failsafe parameter set to a value that will trigger the parachute.
+Commonly you will also need to separately power the parachute servo.
+
+Assuming parachute on MAIN PWM port 7 you might:
 - Connect the parachute to the IO port (MAIN), channel 7 (starting from 1).
 - Power the servo rail - i.e. connect a 5V BEC to the servo rail.
 
-Enable flight termination:
-- Set [Safety](../config/safety.md) action to *Flight termination* for checks where you want the parachute to trigger
-- Set [Failure Detector](../config/safety.md#failure-detector) pitch angles, roll angles and time triggers for crash/flip detection, and disable the failure/IMU timeout circuit breaker (i.e. set [CBRK_FLIGHTTERM=0](../advanced_config/parameter_reference.md#CBRK_FLIGHTTERM)).
-  
-  :::note
-  You can also configure an [external ATS](../config/safety.md#external-automatic-trigger-system-ats) for failure detection.
-  :::
-
-Parachute settings
+To configure the failsafe values for this output in PX4:
 - Set [PWM_MAIN_DIS7](../advanced_config/parameter_reference.md#PWM_MAIN_DIS7) to PWM value for parachute "OFF" position (usually between 700 and 1000ms)
 - Set [PWM_MAIN_FAIL7](../advanced_config/parameter_reference.md#PWM_MAIN_FAIL7) to PWM value for parachute "ON" position (usually between 1800 and 2200ms)
 
-Motor settings:
-- Set [PWM_MAIN_FAILn](../advanced_config/parameter_reference.md#PWM_MAIN_FAIL1), where n is 1 - 4, to 900ms such that the motors directly go to disarmed values.
-
-
 :::note
-There is no way to recover from a Termination state!
-Reboot/power cycle the vehicle before your next test.
+The above values were tested using a spring-loaded launcher from [Fruity Chutes](https://fruitychutes.com/buyachute/drone-and-uav-parachute-recovery-c-21/harrier-drone-parachute-launcher-c-21_33/).
+You will need to use values appropriate for your parachute.
 :::
 
+<!-- 
+The parachute can also be configured using the [Actuators](../config/actuators.md) screen.
+In this case ....
+-->
+
+
+### MAVLink Parachute Setup
+
```

Comment:
@ThomasDebrunner @MaEtUgR Can you please check this section.

I've looked at the code in https://github.com/PX4/PX4-Autopilot/pull/18589 and I want to confirm my understanding

The big picture is clear - the `MAV_CMD_DO_PARACHUTE` is emitted if the parachute is detected and healthy on failsafe. The `COM_PARACHUTE` parameter is checked an will warn if a parachute is expected but not present/healthy.

The small picture is less clear:
-  PX4 appears to base its health checks on getting the heartbeat - that then sets `SYS_STATUS.onboard_control_sensors_present_extended` and `.onboard_control_sensors_health_extended` for the recovery system bit. Is that right?
  -  What about the enabled flag? (`onboard_control_sensors_enabled_extended`) - doesn't seem to be set. I would kind have expected enabled to be based on the `COM_PARACHUTE` parameter.
  - I guess we're assuming that a parachute is always "in a good state". Otherwise we need a mechanism for it to actually tell the flight stack it is OK.
  - What is `COM_PARACHUTE` supposed to do other than be a trigger for sending events when the heartbeat is lost?
- Is the `MAV_CMD_DO_PARACHUTE` sent to the specific component that emits the MAV_TYPE_PARACHUTE? What if two are connected? Or is it just sent to current system ID and component 0 (broadcast).
- Is the `MAV_CMD_DO_PARACHUTE`  command forwarded? i.e. can a GCS trigger the parachute? Should it?

Reply:
Merging as this is, I think correct. I still need the detail in ^^^ and that request is tracked in #1977